pkg/kubelet/kubelet_pods.go

```diff
@@ -88,7 +88,19 @@ func (kl *Kubelet) listPodsFromDisk() ([]types.UID, error) {
 
 // GetActivePods returns non-terminal pods
 func (kl *Kubelet) GetActivePods() []*v1.Pod {
-	allPods := kl.podManager.GetPods()
+	allPods, mirrorPods := kl.podManager.GetPodsAndMirrorPods()
```

Comment:
Why can't we make this change within `GetPodsAndMirrorPods()` fn? I don't see any other place in the code where this is being used.

Reply:
`GetPodsAndMirrorPods()` is defined as:

> // GetPodsAndMirrorPods returns the both regular and mirror pods.

So IMO "regular pods" literally means all pods including "real regular" and "static" pods. I prefer to think of current function as a consistent behavior with the comment.

Reply:
I am fine with the change, just that I am not sure, if there is a need of using static pods somewhere else in the code base instead of using mirror pods. If we have some use then it makes sense to keep the function as is.